src/main/java/org/isf/visits/gui/InsertVisit.java

```diff
@@ -319,49 +308,33 @@ private JButton getButtonOK() {
 		if (buttonOK == null) {
 			buttonOK = new JButton(MessageBundle.getMessage("angal.common.ok.btn"));
 			buttonOK.setMnemonic(MessageBundle.getMnemonic("angal.common.ok.btn.key"));
-			buttonOK.addActionListener(new ActionListener() {
+			buttonOK.addActionListener(actionEvent -> {
```

Comment:
Because this is changed to a lambda the imports for `ActionEvent` and `ActionListener` are no longer needed.

Reply:
Done, thanks!